packages/replay/package.json

```diff
@@ -43,6 +43,7 @@
   "devDependencies": {
     "@types/debug": "^4.1.7",
     "@types/jest": "^28.1.5",
+    "@types/node-fetch": "^2.6.3",
```

Comment:
Unrelated to the fix but this was missing and I didn't notice until I had to rebuild everything.